client/main/ribbon/datatab.js

```diff
@@ -60,18 +60,18 @@ class DataTab extends RibbonTab {
                             new RibbonButton({ title: _('Append'), name: 'appendOutput' })
                         ]})*/
                     ]}),
-                    new RibbonButton({ title: _('Delete'), name: 'delVar', shortcutKey: 'd', shortcutPosition: { x: '25%', y: '75%' } }),
+                    new RibbonButton({ title: _('Delete'), ariaLabel: _('Delete selected variable'), name: 'delVar', shortcutKey: 'd', shortcutPosition: { x: '25%', y: '75%' } }),
                 ]}),
             ]}),
             new RibbonSeparator(),
             new RibbonGroup({ title: _('Rows'), items : [
-                new RibbonButton({ title: _('Filters'), name: 'editFilters', margin: 'large', size: 'large', shortcutKey: 'f', shortcutPosition: { x: '50%', y: '90%' } }),
+                new RibbonButton({ title: _('Filters'), ariaLabel: _('Configure data filters'), name: 'editFilters', margin: 'large', size: 'large', shortcutKey: 'f', shortcutPosition: { x: '50%', y: '90%' } }),
```

Comment:
just `filters`? the user may just want to view them, rather than configure them. nouns rather than verbs?